incubator/mongodb/README.md

```diff
@@ -0,0 +1,192 @@
+# MongoDB Helm Chart
+
+## Prerequisites Details
+* Kubernetes 1.3 with alpha APIs enable
+* PV support on the underlying infrastructure
```

Comment:
We are also using init-containers.  Should we mention this here as well?  That is still in alpha in 1.3?